### DISCUSSION TITLE
Where does the createScheduler come from?

### DISCUSSION THREAD

**Discussion by mrdulin** [ReactiveX/rxjs#7291] [Q&A]
Example: https://rxjs.dev/guide/testing/marble-testing#examples-1

```ts
it('should repeat forever', () => {
  const testScheduler = createScheduler();

  testScheduler.run(({ expectObservable }) => {
    const foreverStream$ = interval(1).pipe(mapTo('a'));

    // Omitting this arg may crash the test suite.
    const unsub = '------!';

    expectObservable(foreverStream$, unsub).toBe('-aaaaa');
  });
});
```

What's `createScheduler`? Where does it come from?

**Comment by jakovljevic-mladen** [comment]
Actually, docs look a bit off here. `createScheduler` could be this:

```ts
const createScheduler = () => new TestScheduler((actual, expected) => {
  expect(actual).deep.equal(expected);
});
```

I copied it [from the first example](https://rxjs.dev/guide/testing/marble-testing#testing-rxjs-code-with-marble-diagrams) from the same page.

If you'd like, please create a PR fixing the docs.
